ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -65,6 +65,9 @@ int enigma_main(int argc, char** argv) {
 
   // Call ENIGMA system initializers; sprites, audio, and what have you
   initialize_everything();
+  
+  // required for some WM's
+  enigma_user::window_center();
```

Comment:
This is entirely missing the point. In fact, `center_size` is a parameter to this function, which has a default argument of false. Therefore, you aren't even doing what it is that you think you are doing.

We are also already calling the function, with the parameter explicitly true, during the room load which occurs after everything is initialized. There is an actual bug in the underlying logic which this is not addressing.
https://github.com/enigma-dev/enigma-dev/blob/fd0923507e710657a54d11956467f4a04b3aeaf1/ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp#L234

Reply:
Interesting. I wondered about that. He mentioned on Discord that the game spawns in a "random corner." What do you suppose is causing that? Also, `window_center` is a PITA because it literally centers the window across however many monitors you have. If you have an even number of monitors, the window will be placed between them. Half on the middle left, half on the middle right.

Reply:
Both you and Josh told me to use window_center() and just that quickly you've changed you mind? Honestly, Josh already told me this change is fine, so I'm not changing anything until I know he's on the same page as you, and I have my doubts.

Reply:
I see, I would suggest actually fixing `window_center` then if it behaves differently from GM.

Finally, I forget how this was fixed for Windows. SDL had the exact same problem also.

Looks like I addressed it in #1468.

Keep in mind this is related to faissaloo's issue #1549.